solutions/swb-app/src/sshKeyRoutes.ts

```diff
@@ -13,9 +13,48 @@ import { isEc2Error } from './errors/ec2Error';
 import { isNoKeyExistsError } from './errors/noKeyExistsError';
 import { isNonUniqueKeyError } from './errors/nonUniqueKeyError';
 import { DeleteSshKeyRequest, DeleteSshKeyRequestParser } from './sshKeys/deleteSshKeyRequest';
+import {
+  ListUserSshKeysForProjectRequest,
+  ListUserSshKeysForProjectRequestParser
+} from './sshKeys/listUserSshKeysForProjectRequest';
 import { SshKeyPlugin } from './sshKeys/sshKeyPlugin';
 
 export function setUpSshKeyRoutes(router: Router, sshKeyService: SshKeyPlugin): void {
+  // List User SSH Keys
+  router.get(
+    '/projects/:projectId/sshKeys',
+    wrapAsync(async (req: Request, res: Response) => {
+      const validatedResult = validateAndParse<ListUserSshKeysForProjectRequest>(
+        ListUserSshKeysForProjectRequestParser,
+        {
+          projectId: req.params.projectId,
+          userId: res.locals.user.id
+        }
+      );
+
+      try {
+        await sshKeyService.listUserSshKeysForProject(validatedResult);
+        res.status(200).send();
+      } catch (e) {
+        console.error(e);
+        if (Boom.isBoom(e)) {
+          throw e;
+        }
+
+        if (isNoKeyExistsError(e)) {
+          throw Boom.notFound(e.message);
+        }
+
+        if (isEc2Error(e) || isAwsServiceError(e) || isNonUniqueKeyError(e)) {
```

Comment:
take out the is non unique key error since we will allow since this is a list call